packages/frameworks-express/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@auth/express",
+  "description": "Authentication for Express.",
+  "version": "0.0.1",
+  "type": "module",
+  "files": [
+    "index.*",
+    "http-api-adapters.*",
+    "src"
+  ],
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js"
+    },
+    "./http-api-adapters": {
+      "types": "./http-api-adapters.d.ts",
+      "import": "./http-api-adapters.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "tsup --config ./tsup.config.js && node scripts/postbuild",
+    "patch": "npm version patch --no-git-tag-version",
+    "clean": "rm -rf index.*",
```

```suggestion
    "clean": "rm -rf lib index.*",
```

See https://github.com/nextauthjs/next-auth/pull/8354/files#r1302249740